Shared/Views/RelistenMenuView.swift

```diff
@@ -260,7 +261,7 @@ public class RelistenMenuView : UIView {
         case .DiscoverRandom: return "random"
             
         case .RecentlyPlayed: return "recently played"
-        case .RecentlyAdded: return "recently performed"
+        case .RecentlyAdded: return "recently added"
```

Comment:
Not sure if you agree with this name change- feel free to revert it. It felt awkward looking at the recently performed shows for the Grateful Dead, and it looks weird when shows are listed out of performance order because a source was updated.

Reply:
maybe we should have both?

Reply:
Good idea. https://github.com/orgs/RelistenNet/projects/1#card-11616883